cli/src/main/java/com/devonfw/tools/ide/io/FileAccessImpl.java

```diff
@@ -399,12 +406,39 @@ private void deleteLinkIfExists(Path path) throws IOException {
 
     boolean isJunction = isJunction(path); // since broken junctions are not detected by Files.exists()
     boolean isSymlink = Files.exists(path) && Files.isSymbolicLink(path);
+    
+    // Also check for broken symlinks that exist but don't resolve (Files.exists() returns false for these)
+    if (!isSymlink && Files.exists(path, LinkOption.NOFOLLOW_LINKS) && Files.isSymbolicLink(path)) {
+      isSymlink = true; // This is a broken symlink
+    }
```

Comment:
Why not fixing the actual problem directly at the root?
```suggestion
    boolean isSymlink = Files.exists(path, LinkOption.NOFOLLOW_LINKS) && Files.isSymbolicLink(path);
```